src/Sulu/Bundle/AdminBundle/Resources/js/components/Toolbar/Controls.js

```diff
@@ -1,17 +1,51 @@
 // @flow
-import type {Node} from 'react';
+import type {ChildrenArray} from 'react';
 import React from 'react';
-import toolbarStyles from './toolbar.scss';
+import classNames from 'classnames';
+import type {Group, Item, Skins} from './types';
+import controlsStyles from './controls.scss';
 
 type Props = {
-    children: Node,
+    children: ChildrenArray<Item | Group | false>,
+    skin?: Skins,
 };
 
 export default class Controls extends React.PureComponent<Props> {
+    static defaultProps = {
+        skin: 'light',
+    };
+
+    static createChildren(children: ChildrenArray<Item | Group | false>, skin?: Skins) {
+        return React.Children.map(children, (child: Item | Group | false) => {
+            if (!child) {
+                return;
+            }
+
+            // $FlowFixMe
```

Comment:
I have created a issue in the flow repository for that, let's see if they tackle it :-) https://github.com/facebook/flow/issues/5934